src/zzz_od/operation/enter_game/open_game.py

```
@@ -45,12 +46,15 @@ def open_game(self) -> OperationRoundResult:
         log.info('命令行指令 %s', command)
 
-        # CREATE_BREAKAWAY_FROM_JOB:启动器用进程组管理时,使子进程逃离 jobobject,
-        # 避免 OneDragon-Launcher.exe 退出后游戏被杀死。
-        subprocess.Popen(
-            command,
-            creationflags=subprocess.CREATE_BREAKAWAY_FROM_JOB
-        )
+        if not getattr(sys, 'frozen', False):
+            # CREATE_BREAKAWAY_FROM_JOB:启动器用进程组管理时,使子进程逃离 jobobject,
+            # 避免 OneDragon-Launcher.exe 退出后游戏被杀死。
+            subprocess.Popen(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)